test/local/account_routes.js

```diff
@@ -701,6 +701,20 @@ test('/account/login', function (t) {
       })
     })
 
+    t.test('location data is present in sign-in confirmation email', function (t) {
+      config.signinConfirmation.sample_rate = 1
+      return runTest(route, mockRequest, function (response) {
+        t.equal(mockMailer.sendVerifyLoginEmail.callCount, 1, 'mailer.sendVerifyLoginEmail was called')
+        t.equal(mockMailer.sendVerifyLoginEmail.getCall(0).args[2].location.city, 'Mountain View')
```

Comment:
In the same note, should add tests that verify that no location data is present in the `New Device Login Notification` if the `GEO_ENABLED = false`